meta-cube/recipes-core/lxc/files/lxc-overlayclean

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh -
```

Comment:
It isn't clear from the commit message why we don't need this any more. Can you elaborate on why it is dropped ?